simple_navigation_node:
  ros__parameters:

    vx_trace: 0.6
    wz_trace: 1.0
    wz_turning: 0.5

    slowdown_dist: 1.0
    vx_slowdown: 0.2
    accel_period: 3.0

    min_turning_angle: 6.0
    goal_dist_thresh: 0.2
    hdg_diff_thresh: 1.0

    turn_p: 0.01
    turn_i: 0.0001
    turn_d: 0.001

    steer_p: 0.01
    steer_i: 0.0
    steer_d: 0.0